recipe/meta.yaml

```diff
@@ -19,17 +19,18 @@ build:
 
 requirements:
   host:
-    - python >=3.6
```

Comment:
please add a skip for python variant less 3.6 ( skip: True. # [py<36] )

Reply:
Done